pkg/cmd/openshift-sdn/sdn.go

```diff
@@ -66,3 +62,18 @@ func (sdn *OpenShiftSDN) initSDN() error {
 func (sdn *OpenShiftSDN) runSDN() error {
 	return sdn.OsdnNode.Start()
 }
+
+func (sdn *OpenShiftSDN) writeConfigFile() error {
+	// Make an event that openshift-sdn started
+	sdn.sdnRecorder.Eventf(&kclientv1.ObjectReference{Kind: "Node", Name: sdn.NodeConfig.NodeName}, kclientv1.EventTypeNormal, "Starting", "Starting openshift-sdn.")
```

Comment:
I would change this this wording to something like "openshift-sdn done initializing node networking"